ubuntu1910/Dockerfile

```diff
@@ -0,0 +1,120 @@
+FROM ubuntu:19.10
+
+LABEL description="Ubuntu 19.10 with Acts dependencies"
+LABEL maintainer="Paul Gessinger <paul.gessinger@cern.ch>"
+# increase whenever any of the RUN commands change
+LABEL version="1"
+
+# install system dependencies
+#
+# see also https://root.cern.ch/build-prerequisites
+# DEBIAN_FRONTEND ensures non-blocking operation (tzdata is a problem)
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y update \
+  && apt-get -y install \
+    build-essential \
+    cmake \
+    curl \
+    doxygen \
+    git \
+    libboost-dev \
+    libboost-filesystem-dev \
+    libboost-program-options-dev \
+    libboost-test-dev \
+    libeigen3-dev \
+    libexpat-dev \
+    libfftw3-dev \
+    libjpeg-dev \
+    libpng-dev \
+    libtbb-dev \
+    libx11-dev \
+    libxext-dev \
+    libxft-dev \
+    libxpm-dev \
+    libxerces-c-dev \
+    ninja-build \
+    python3 \
+    python3-dev \
+    python3-pip \
+    rsync \
+    xsltproc \
+    zlib1g-dev \
+  && apt-get -y clean
+
+# install external dependencies
+ENV GET curl --location --silent --create-dirs --output
+ENV PREFIX /usr/local
+
+# ROOT
+RUN ${GET} src/root.tar.gz \
+    https://root.cern/download/root_v6.20.04.source.tar.gz \
+  && tar -xf src/root.tar.gz --strip-components=1 --directory src \
```

Comment:
I don't think you're deleting the `tar.gz` file. The line that I had (`curl -SL URL | tar -xzC .`) directly untars and never creates the tar file.

Reply:
The tar file is downloaded to the `src` directory which is deleted at the end. I wanted to add checksum testing at some point but never did. That is why I download the file to disk first.